runtime/commands/fun.js

```diff
@@ -635,4 +638,115 @@ Commands.shorten = {
   }
 }
 
+Commands.price = {
+  name: 'price',
+  help: 'Current price from coinmarketcap.com !',
+  aliases: ['price'],
+  timeout: 5,
+  level: 0,
+  fn: function (msg, suffix) {
+    var tags = suffix.split(/\s+/)
+    var fr = ''
+    var from = ''
+    var template = {
+      'title': 'Full name of Coin',
+      'description': 'Symbol',
+      'color': 431042,
+      'thumbnail': {
+        'url': 'https://files.coinmarketcap.com/static/img/coins/32x32/'
+      },
+      'image': {
+        'url': ''
+      },
+      'author': {
+        'name': 'CoinMarketCap',
+        'url': 'http://coinmarketcap.com',
+        'icon_url': 'https://coinmarketcap.com/static/img/CoinMarketCap.png'
+      },
+      'fields': []
+    }
+
+    if (typeof tags[0] === 'undefined' || tags[0] === '') {
```

Comment:
This would be easier to accomplish with `if (!tags) {}` or even `if (!tags[0]) {}`.

Reply:
I think there was a reason for checking just for undefined and empty string, otherwise I agree.